index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id= "navbar" style ="margin-top: 25px;">
+      <ul>
+        <li><header><a href=#Introduction>Introduction</a></header></li>
+        <li><a class="nav-link" href=#History>History</a></li>
+        <li><a class="nav-link" href=#Archeticture>Archeticture</a></li>
+        <li><a class="nav-link" href=#Blockchain>Blockchain</a></li>
+        <li><a class="nav-link" href=#Nodes>Nodes</a></li>
+        <li><a class="nav-link" href=#Mining>Mining</a></li>
+        <li><a class="nav-link" href=#Wallets>Wallets</a></li>
+        <li><a class="nav-link" href=#Economics>Economics</a></li>
+      </ul>
+    </nav>
+    <main id="main-doc" style="margin-left:25%;padding:1px 16px;height:1000px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in a separate CSS file.